src/applications/facility-locator/api/index.js

```diff
@@ -4,7 +4,13 @@ import environment from 'platform/utilities/environment';
 
 // To use vets-api data locally, replace the following with this:
 // export default LiveApi;
-export default ((environment.isLocalhost() || window.Cypress) &&
-!environment.API_URL.includes('review.vetsgov')
-  ? MockApi
-  : LiveApi);
+
+const getAPI = () => {
+  return !window.Mocha &&
+    (environment.isLocalhost() || window.Cypress) &&
```

Comment:
Would it be possible to convert this ternary into if statements? it's getting a bit difficult to read the logic